bitnami/clickhouse/README.md

```diff
@@ -593,6 +595,17 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 6.0.0
```

Comment:
I would add a note regarding the bump of the `bitnami/zookeper` major

Reply:
Done